neo/Consensus/ConsensusService.cs

```diff
@@ -537,8 +538,13 @@ private void RequestChangeView()
             byte expectedView = context.ChangeViewPayloads[context.MyIndex]?.GetDeserializedMessage<ChangeView>().NewViewNumber ?? 0;
             if (expectedView < context.ViewNumber) expectedView = context.ViewNumber;
             expectedView++;
-            Log($"request change view: height={context.BlockIndex} view={context.ViewNumber} nv={expectedView}");
             ChangeTimer(TimeSpan.FromSeconds(Blockchain.SecondsPerBlock << (expectedView + 1)));
+            if ((context.CountCommitted() + context.CountFailed()) > context.F())
+            {
+                Log($"Node will not request change view to nv={expectedView} because nc={context.CountCommitted()}+nf={context.CountFailed()} > {context.F()}");
```

Comment:
Suggest shorter working `Not requesting change view to ...`

Reply:
Thanksss, Jeff the Magister.